cypress/integration/team/teams_spec.js

```diff
@@ -107,4 +107,36 @@ describe('Teams Suite', () => {
         // 9. Remove user from team
         cy.removeTeamMember(teamURL, user.firstName);
     });
+
+    it('TS14633 Leave all teams', () => {
+        // 1. Login and go to /
+        cy.apiLogin('user-2');
+        cy.visit('/');
```

Comment:
Though this spec is mean to run once and on fresh data, this will fail on retries(on pipeline or local dev).  I'd suggest to create a new team before leaving all team.  Something like,
```suggestion
        cy.apiCreateTeam('test-team', 'Test Team').then(() => {
            cy.visit('/');
        });
```